### PR TITLE
PB Updates for HLL Datatypes

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
### Open questions?

- Merge order?
- Should do go against develop or another branch (develop is most up to date)
- Dealing w/ PB changes... AGAIN.

### CODE REVIEW

File: src/riak_dt.proto [hunk 1]
```diff
@@ -107,6 +108,11 @@ message DtValue {
     optional sint64   counter_value = 1;
     repeated bytes    set_value     = 2;
     repeated MapEntry map_value     = 3;
+
+    /* We return an estimated cardinality of the Hyperloglog set
+     * on fetch.
+     */
+    optional sint64   hll_value     = 4;
```
Comment (russelldb) [inline src/riak_dt.proto]: why `sint64`? I thought hll cards were always positive

File: src/riak_dt.proto [hunk 2]
```diff
@@ -251,4 +271,5 @@ message DtUpdateResp {
     optional sint64   counter_value = 3;
     repeated bytes    set_value     = 4;
     repeated MapEntry map_value     = 5;
-}
\ No newline at end of file
+    optional sint64   hll_value     = 6;
```
Comment (russelldb) [inline src/riak_dt.proto]: same question as above re type `sint64`

File: test/bucket_props_codec_eqc.erl [hunk 3]
```diff
@@ -137,7 +137,7 @@ linkfun() ->
     ?LET({M,F}, {atom(), atom()}, {linkfun, {modfun, M, F}}).
 
 datatype() ->
-    {datatype, elements([counter, set, map])}.
+    {datatype, elements([counter, set, map, hll])}.
```
Comment (russelldb) [inline test/bucket_props_codec_eqc.erl]: Had no idea we had this duplicate list here. I guess we can't depend on riak_kv_types for this. Eugh. OK.

**Comment by russelldb** [comment]
+1 from me. 

Nag nag about mixing formatting and functional changes.

**Comment by russelldb** [comment]
+1

File: src/riak_dt.proto [hunk 4]
```diff
@@ -22,7 +22,8 @@
 */
 
 /*
-** Revision: 2.0
+%% TODO: Set for whichever release version we use
+** Revision: 2.3.0
```
Comment (nickelization) [inline src/riak_dt.proto]: Should we modify this to 2.2.0?

Comment (zeeshanlakhani) [inline src/riak_dt.proto]: I left this question to @JeetKunDoug w/ the release time (we didn't know at the time)... so @nickelization, you're good to do that (and keep it as a separate commit) if Riak team is ok w/ it ;).

**Comment by nickelization** [comment]
:+1: 84ec1f4

**Comment by nickelization** [comment]
Build was successful, but Bors seems to be stuck for some reason. Going ahead and merging this manually.
